test/postgresql.order.test.js

```diff
@@ -0,0 +1,228 @@
+// Copyright IBM Corp. 2020. All Rights Reserved.
+// Node module: loopback-connector-postgresql
+// This file is licensed under the Artistic License 2.0.
+// License text available at https://opensource.org/licenses/Artistic-2.0
+
+'use strict';
+
+require('./init');
+const should = require('should');
+const async = require('async');
+
+let db,
+  PostWithDefaultIdSort,
+  PostWithDisabledDefaultIdSort,
+  PostWithNumericDefaultIdSort,
+  PostWithNumericStringDefaultIdSort;
+
+describe('Order settings', function() {
+  before(function() {
+    db = global.getSchema();
+  });
+
+  before(function(done) {
+    PostWithDefaultIdSort = db.define(
+      'PostWithDefaultIdSort',
+      {
+        id: {type: Number, id: true},
+        title: {type: String, length: 255, index: true},
+        content: {type: String},
+      },
+      {
+        defaultIdSort: true,
+      },
+    );
+
+    PostWithDisabledDefaultIdSort = db.define(
+      'PostWithDisabledDefaultIdSort',
+      {
+        id: {type: Number, id: true},
+        title: {type: String, length: 255, index: true},
+        content: {type: String},
+      },
+      {
+        defaultIdSort: false,
+      },
+    );
+
+    PostWithNumericDefaultIdSort = db.define(
+      'PostWithNumericDefaultIdSort',
+      {
+        id: {type: Number, id: true},
+        title: {type: String, length: 255, index: true},
+        content: {type: String},
+      },
+      {
+        defaultIdSort: 'numericIdOnly',
+      },
+    );
+
+    PostWithNumericStringDefaultIdSort = db.define(
+      'PostWithNumericStringDefaultIdSort',
+      {
+        id: {type: String, id: true},
+        title: {type: String, length: 255, index: true},
+        content: {type: String},
+      },
+      {
+        defaultIdSort: 'numericIdOnly',
+      },
+    );
+
+    done();
+  });
+
+  before(function(done) {
+    db.automigrate([
+      'PostWithDefaultIdSort',
+      'PostWithDisabledDefaultIdSort',
+      'PostWithNumericDefaultIdSort',
+      'PostWithNumericStringDefaultIdSort'],
+    done);
+  });
+
+  after(function(done) {
+    async.parallel([
+      cb => PostWithDefaultIdSort.destroyAll(cb),
+      cb => PostWithDisabledDefaultIdSort.destroyAll(cb),
+      cb => PostWithNumericDefaultIdSort.destroyAll(cb),
+      cb => PostWithNumericStringDefaultIdSort.destroyAll(cb),
+    ], done);
+  });
+
+  it('builds base query', function(done) {
+    const res = db.connector.buildSelect('PostWithDefaultIdSort', {});
+    const sql = res.sql;
+
+    sql.should.be.equal('SELECT "id","title","content" FROM "public"."postwithdefaultIdSort" ORDER BY "id"');
+    done();
+  });
+
+  it('builds non-ordering base query', function(done) {
+    const res = db.connector.buildSelect('PostWithDisabledDefaultIdSort', {});
+    const sql = res.sql;
+
+    sql.should.be.equal('SELECT "id","title","content" FROM "public"."postwithdisableddefaultIdSort"');
```

Comment:
The tests are failing now because `postwithdisableddefaultIdSort` should be `postwithdisableddefaultidsort`(similar thing for the below tests).

Reply:
Thanks, I adjusted the casing as suggested.